src/arrayMethodSort.js

```diff
@@ -6,6 +6,34 @@
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
     // write code here
+    let temp = 0;
+    let flag = true;
+
+    while (flag) {
+      flag = false;
+
+      if (compareFunction === undefined) {
+        for (let i = 0; i < this.length - 1; i++) {
+          if (this[i] + '' > this[i + 1] + '') {
```

Comment:
better for reading, as for me
```suggestion
          if (String(this[i]) > String(this[i + 1])) {
```